starfyre/dom_methods.py

```diff
@@ -91,19 +91,47 @@ def is_attribute(name):
     if not component.is_text_component:
         html += f"{prop_string} >"
 
+    inner_content = component.inner_content
+    inner_html = ""
+    inner_css = ""
+    inner_js = ""
+
+    for innerElement in inner_content:   
+        innerElement.parentElement = component
+        new_html, new_css, new_js = render_helper(innerElement)
+        inner_html += new_html
+        inner_css += new_css
+        inner_js += new_js
+
+    is_slot_used = False
```

Comment:
This should be a property in the component class only. That way we can identify if the component is a slot component or not.